prometheus/config/prometheus.yml

```diff
@@ -1,6 +1,6 @@
 global:
   # TODO This should be increased before release
```

Comment:
I say lets remove this TODO.  We can always adjust this value later, but `10m` feels like the best place to start.

Can you also update the prometheus `scrapeInterval` value in `grafana/provisioning/datasources/datasource.yml`?  Originally I was thinking that should match this scrape interval, but it seems like in the worst case that would mean the data in Grafana would be `10m` old compared to the data in Prometheus (which itself could be `10m` old).  So, unless we start bogging down the Prometheus instance somehow, I am thinking it would be better for Grafana to hit Prometheus every `1m`.  (Also, I realized when I tested just now that the `scrapeInterval` value in the `datasource.yml` does not seem to be used.  According to the [Documentation](https://grafana.com/docs/grafana/latest/administration/provisioning/#json-data) we should be setting:

```yml
datasources:
  - name: Prometheus
    ...
    jsonData:
      timeInterval: 1m
```